soc/intel/intel_adsp/ace/Kconfig.defconfig.series

```diff
@@ -50,7 +50,7 @@ config INTEL_ADSP_TIMER
 config DYNAMIC_INTERRUPTS
 	default y
 
-if LOG
+if LOG && !SOF
```

Comment:
we shoould not be using SOF symbols here. SOF is not part of Zephyr.

Reply:
@nashif sorry, I guess I'm missing something. The `SOF` Kconfig symbol is defined in Zephyr in https://github.com/zephyrproject-rtos/zephyr/blob/bb699ff2df249addb9473d0bbd2f6949f008161e/modules/Kconfig.sof#L7 Similarly other modules define their symbols in modules/Kconfig.*, e.g. https://github.com/zephyrproject-rtos/zephyr/blob/bb699ff2df249addb9473d0bbd2f6949f008161e/modules/Kconfig.esp32#L7 which is then used in https://github.com/zephyrproject-rtos/zephyr/blob/bb699ff2df249addb9473d0bbd2f6949f008161e/drivers/wifi/esp32/Kconfig.esp32#L7 . Are those cases wrong too?